src/ServiceControl/Infrastructure/RavenDB/Expiration/ExpirySagaAuditIndex.cs

```diff
@@ -15,16 +15,13 @@ public class Result
 
         public ExpirySagaAuditIndex()
         {
-            Map = (sagaHistories => from sagaHistory in sagaHistories
+            Map = sagaHistories => from sagaHistory in sagaHistories
                 select new Result
                 {
-                    LastModified = MetadataFor(sagaHistory)["Last-Modified"],
-                });
+                    LastModified = MetadataFor(sagaHistory).Value<DateTime>("Last-Modified").Ticks
+                };
 
             DisableInMemoryIndexing = true;
-
-            Sort(result => result.LastModified, SortOptions.String);
```

Comment:
Should there still be a sort on this index? When we read the index we apply a sort to the query so it will need to get sorted anyway.